Utilities/XrdAdaptor/src/QualityMetric.cc

```diff
@@ -157,16 +157,17 @@ QualityMetricFactory * QualityMetricFactory::m_instance = new QualityMetricFacto
 std::unique_ptr<QualityMetricSource>
 QualityMetricFactory::get(timespec now, const std::string &id)
 {
-    MetricMap::const_iterator it = m_instance->m_sources.find(id);
```

Comment:
Ah - I struggled to find an explanation of the differences and performance considerations between `concurrent_unordered_map` and `concurrent_hash_map`.  Semantically, either should work in this case.  What's the advantage of the above?